.envrc

```diff
@@ -1,8 +1,6 @@
 if $(command -v asdf > /dev/null)
 then
-	GOVERSION=$(asdf current golang | awk '{print $2}')
-	export GOROOT=$HOME/.asdf/installs/golang/$GOVERSION/go
-	PATH_add $HOME/.asdf/installs/golang/$GOVERSION/packages/bin
+    PATH_add ${HOME}/.asdf/shims
```

Comment:
this should already be done as part of asdf, if they're not on the path there are other issues here.

Reply:
then we dont need the above

Reply:
hmm seems that shims are not added when you brew install :/
but anyway Ive updated to use the go version that should be set in the shims (as you mentioned should be in the asdf install to set the GOROOT like the GOPATH too)

Reply:
yeah well there should be no asdf stuff in here tbh ... Ive refactored to remove all asdf stuff.